DesktopEdge/Models/ZitiService.cs

```diff
@@ -121,13 +121,33 @@ public class ZitiService {
 
 		public Visibility TimerVisibility {
 			get {
-				return (TimeoutRemaining>-1&&TimeoutRemaining<1260) ? Visibility.Visible : Visibility.Collapsed;
+				return (TimeoutCalculated > -1&& TimeoutCalculated < 1260&& TimeoutCalculated > 0) ? Visibility.Visible : Visibility.Collapsed;
+			}
+			set { }
+		}
+
+		public int TimeoutCalculated { 
+			get {
+				if (this.TimeoutRemaining == -1|| TimeoutRemaining == 0) return this.TimeoutRemaining;

```

Comment:
```
this.TimeoutRemaining == -1|| TimeoutRemaining == 0
```
could be simplified by just using `TimeoutRemaining <= 0`

Reply:
No because -1 means it isnt set, 0 means it timed out and I have set at some point -300 to show a negative time